aws/k8s-vault-aws-start.sh

```diff
@@ -3,6 +3,20 @@
 # set -o pipefail
 # set -o nounset
 
+function checkCommandsAvailable() {
+  for var in "$@"
+  do
+    if ! [ -x "$(command -v "$var")" ]; then
+      echo "🔥 ${var} is not installed." >&2
+      exit 1
+    else
+      echo "🏄 $var is installed..."
+    fi
+  done
+}
+
+checkCommandsAvailable helm minikube jq vault sed grep docker grep cat
```

Comment:
```suggestion
checkCommandsAvailable helm minikube jq vault sed grep docker grep cat aws
```

Reply:
Vault is necessary, the script uses:

`vault kv put secret/secret-challenge`